src/components/Fields/MarkdownLabel.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+import PropTypes from 'prop-types';
+import { ALLOWED_MARKDOWN_LABEL_TAGS } from '../../utils/config';
+
+const MarkdownLabel = ({ label }) => (
```

Comment:
I'd really like to see this component be able to do more than just wrap `ReactMarkdown`. 

It looks like almost all instances of field labels wrap the label in a `<h4>`, and most (though not all) also pass a className. Can we update the component to use these at a minimum?

Reply:
So to elaborate further, the _best_ case would be to replace all the duplicate CSS for labels with a single class for the `MarkdownLabel` component. Assuming that some inputs still need specific (and different) CSS, we could then pass through any additional classes.

Reply:
> I'd really like to see this component be able to do more than just wrap `ReactMarkdown`.
> 
> It looks like almost all instances of field labels wrap the label in a `<h4>`, and most (though not all) also pass a className. Can we update the component to use these at a minimum?

I've noticed that you also implemented this for option labels for things like checkboxes and the radio group items, which makes the above more complicated. 

Hopefully you can still work this out, but let me know if not.

Reply:
Fleshed this out more. I ended up with two labels with overlap and a couple extra special cases.